functorch/_src/eager_transforms.py

```diff
@@ -173,7 +174,73 @@ def wrapper(cotangents, retain_graph=True, create_graph=None):
 
     return results, wrapper
 
-def jacrev(f, argnums=0):
+
+def jacrev(f: Callable, argnums: Union[int, Tuple[int]] = 0):
+    """
+    Computes the Jacobian of :attr:`f` with respect to the arg(s) at index :attr:`argnum` using reverse
+    mode autodiff
+
+    Args:
+        func (function): A Python function that takes one or more arguments, one of which must be a
+            Tensor, and returns one or more Tensors
+        argnums: Optional, integer or tuple of integers, saying which arguments to get the Jacobian with
+            respect to. Default: 0.
+
+    Returns:
+        Returns a function that takes in the same inputs as :attr:`f` and returns the Jacobian of :attr:`f` with 
+        respect to the arg(s) at :attr:`argnums`
+
+    A basic usage with a pointwise, unary operation will give a diagonal array as the Jacobian
+        >>> from functorch import jacrev
+        >>> x = torch.randn(5)
+        >>> jacobian = jacrev(torch.sin)(x)
+        >>> expected = torch.diag(torch.cos(x))
+        >>> assert torch.allclose(jacobian, expected)
+
+    :func:`jacrev` can be composed with vmap to produce batched Jacobians:
+        >>> from functorch import jacrev
+        >>> x = torch.randn(64, 5)
+        >>> jacobian = vmap(jacrev(torch.sin))(x)
+        >>> assert jacobian.shape == (64, 5, 5)
+    
+    By default, :func:`jacrev` computes the Jacobian with respect to the first input. However, it can compute the
+    Jacboian with respect to a different argument by using :attr:`argnums`:
+        >>> from functorch import jacrev
+        >>> def f(x, y):
+        >>>   return x + y ** 2
+        >>>
+        >>> x, y = torch.randn(5), torch.randn(5)
+        >>> jacobian = jacrev(f, argnums=1)(x, y)
+        >>> expected = torch.diag(2 * y)
+        >>> assert torch.allclose(jacobian, expected)
+    
+    Additionally, passing a tuple to :attr:`argnums` will compute the Jacobian with respect to multiple arguments
+        >>> from functorch import jacrev
+        >>> def f(x, y):
+        >>>   return x + y ** 2
+        >>>
+        >>> x, y = torch.randn(5), torch.randn(5)
+        >>> jacobian = jacrev(f, argnums=(0,1))(x, y)
+        >>> expectedX = torch.diag(torch.ones_like(x))
+        >>> expectedY = torch.diag(2 * y)
+        >>> assert torch.allclose(jacobian[0], expectedX)
+        >>> assert torch.allclose(jacobian[1], expectedY)
+
+    .. note:
```

Comment:
Since the function calls vmap and grad, figured this note also applied to `jacrev`